var/spack/repos/builtin/packages/blis/package.py

```diff
@@ -111,8 +111,12 @@ def darwin_fix(self):
 
     @property
     def libs(self):
+        shared = True if '+shared' in self.spec else False
         return find_libraries(
-            ["libblis", "libblis-mt"], root=self.prefix, recursive=True
+            ["libblis", "libblis-mt"],
+            root=self.prefix,
+            shared=shared,
```

Comment:
Or you could simply do the following:

```suggestion
            shared='+shared' in self.spec,
```

Reply:
Incorporated review comments